common/changes/@bentley/geo-tools-react/successful_warnings_2021-07-08-18-30.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@bentley/geo-tools-react",
+      "comment": "add rebuild script",
+      "type": "none"
```

Comment:
@calebmshafer i had to manually modify these files to set the type to none because the only options available were path or minor. 
Will this be an issue?

Reply:
Making it none will cause it not to bump the version of the package.